core/src/trezor/ui/layouts/lvgl/__init__.py

```
@@ -1338,6 +1339,26 @@
+async def confirm_sol_message(
+    ctx: wire.GenericContext, address: str, app_domain_fd: str | None, message: str
+) -> None:
+    from trezor.lvglui.scrs.template import Message
+
+    screen = Message(
+        _(i18n_keys.TITLE__SIGN_STR_MESSAGE).format("SOL"),
+        address,
+        message,
+        ctx.primary_color,
+        ctx.icon_path,
+        False,
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)